eq-author-api/schema/resolvers/base.js

```diff
@@ -206,7 +206,11 @@ const Resolvers = {
       const questionnareComments = await getCommentsForQuestionnaire(
         questionnaireId
       );
-      return questionnareComments.comments[id] || [];
+      let sortedComments = questionnareComments.comments[id] || [];
+      sortedComments = sortedComments.sort(
```

Comment:
If you accept Matt's suggestion you'll need to drop the re-assignment. As he says, it sorts in place:

```suggestion
      sortedComments.sort(
```